src/main/java/org/jabref/gui/menus/RightClickMenu.java

```diff
@@ -102,6 +103,7 @@ public RightClickMenu(JabRefFrame frame, BasePanel panel) {
 
         add(new GeneralAction(Actions.SEND_AS_EMAIL, Localization.lang("Send as email"), IconTheme.JabRefIcon.EMAIL.getSmallIcon()));
         addSeparator();
+        add(new ExportLinkedFilesAction());
```

Comment:
Do you really think that this export functionality is used that often that we should add it to the right click menu?

Reply:
It was requested by @koppor  And I think it makes sense to have it in the context menu